src/devtools-connector/devtools-arc-inspector.ts

```diff
@@ -69,6 +73,13 @@ class DevtoolsArcInspector implements ArcInspector {
         }
       });
 
+      if (!this.arc.isSpeculative) {
+        this.arcDevtoolsChannel.send({
+          messageType: 'register-arc',
```

Comment:
Changed! But somehow we need the onRawMessageBundle() function to listen to this message type.

Reply:
This is because you only receives messages for the arc selected in the UI. If you have a global component (as you do here) onRawMessageBundle is the way to go :) Nice find!